src/test/java/usdot/fhwa/stol/c2c/c2c_mvt/controllers/TestStandardValidationController.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package usdot.fhwa.stol.c2c.c2c_mvt.controllers;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+
+/**
+ *
+ * @author Federal Highway Administration
```

Comment:
Please use developer name

Reply:
Removed this file. StandardValidationControllerTest.java is the class that contains the tests Eric developed